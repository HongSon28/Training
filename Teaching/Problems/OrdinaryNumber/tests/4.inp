161
846921
947937
839355
949405
252593
400978
325101
561449
605970
1293
133436
834046
247757
173231
507618
344192
583733
649297
404349
167981
281901
108699
605501
422059
115227
779192
368540
604861
999961
752183
722545
9038
105633
852521
196770
929966
935041
484393
469340
536345
261128
907273
348017
672481
874469
768739
698887
97671
251577
752529
254181
165621
754553
979409
404567
881026
966273
571309
961039
676163
692285
841025
75315
464564
49643
24689
20825
287009
968673
365523
692167
467233
298809
409431
780832
923027
275136
325113
30021
232249
416081
504361
20369
986721
768378
87439
21465
518415
61389
193507
268493
146689
247241
519121
350834
720557
263601
70149
934546
26389
473372
518389
569441
575658
786563
175081
605025
14872
908986
947660
655781
767661
870972
647151
104531
687917
376051
292756
632018
507264
681253
741469
236401
425409
54461
472818
863206
476768
608193
828199
93256
190347
515297
921436
193705
55436
847583
593243
687549
933744
359321
339701
149929
247903
574897
45669
845904
251227
582445
97751
188937
595898
447859
710937
391301
484529
217673
979271
136343
681345
474866
