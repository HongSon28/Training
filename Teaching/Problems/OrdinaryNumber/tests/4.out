52
53
52
53
47
48
47
50
50
28
46
52
47
46
49
48
50
50
48
46
47
45
50
48
46
52
48
50
53
51
51
35
45
52
46
53
53
49
49
49
47
53
48
51
52
51
51
44
47
51
47
46
51
53
48
52
53
50
53
51
51
52
42
49
40
38
37
47
53
48
51
49
47
48
52
53
47
47
38
47
48
49
37
53
51
43
37
49
41
46
47
46
47
49
48
51
47
42
53
38
49
49
50
50
52
46
50
37
53
53
50
51
52
50
45
51
48
47
50
49
51
51
47
48
40
49
52
49
50
52
44
46
49
53
46
40
52
50
51
53
48
48
46
47
50
40
52
47
50
44
46
50
49
51
48
49
46
53
46
51
49
